seed_data/lib_test.py

```diff
@@ -0,0 +1,73 @@
+"""
+Tests for library functions used by seed_db and alter_data commands
+"""
+
+from django.test import TestCase
+from micromasters.factories import UserFactory
+from courses.factories import ProgramFactory, CourseFactory, CourseRunFactory
+from seed_data.lib import (
+    CourseFinder,
+    CourseRunFinder,
+    UserFinder,
+)
+
+
+class UserFinderTests(TestCase):
+    """Test cases for UserFinder"""
+    @classmethod
+    def setUpTestData(cls):
+        cls.user = UserFactory.create(username='username1', email='email1@example.com')
+        UserFactory.create(username='username2', email='email2@example.com')
+
+    def test_user_finder_success(self):
+        """Tests that UserFinder will return a desired user"""
+        found_users = []
```

Comment:
Nit: I think this would be a little cleaner if you moved all the finds into the literal instead of appending. Same for the other tests here.